Gemfile.lock

```diff
@@ -1,57 +1,57 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    actionpack (5.2.1)
-      actionview (= 5.2.1)
-      activesupport (= 5.2.1)
+    actionpack (5.2.2.1)
+      actionview (= 5.2.2.1)
+      activesupport (= 5.2.2.1)
       rack (~> 2.0)
       rack-test (>= 0.6.3)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.0, >= 1.0.2)
-    actionview (5.2.1)
-      activesupport (= 5.2.1)
+    actionview (5.2.2.1)
+      activesupport (= 5.2.2.1)
       builder (~> 3.1)
       erubi (~> 1.4)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.0, >= 1.0.3)
-    activemodel (5.2.1)
-      activesupport (= 5.2.1)
-    activerecord (5.2.1)
-      activemodel (= 5.2.1)
-      activesupport (= 5.2.1)
+    activemodel (5.2.2.1)
+      activesupport (= 5.2.2.1)
+    activerecord (5.2.2.1)
+      activemodel (= 5.2.2.1)
+      activesupport (= 5.2.2.1)
       arel (>= 9.0)
-    activesupport (5.2.1)
+    activesupport (5.2.2.1)
       concurrent-ruby (~> 1.0, >= 1.0.2)
       i18n (>= 0.7, < 2)
       minitest (~> 5.1)
       tzinfo (~> 1.1)
     arel (9.0.0)
     builder (3.2.3)
-    concurrent-ruby (1.1.1)
+    concurrent-ruby (1.1.5)
     crass (1.0.4)
     double_entry (1.0.1)
       activerecord (>= 3.2.0)
       activesupport (>= 3.2.0)
       money (>= 6.0.0)
       railties (>= 3.2.0)
-    erubi (1.7.1)
-    i18n (1.1.1)
+    erubi (1.8.0)
+    i18n (1.6.0)
       concurrent-ruby (~> 1.0)
-    json (2.1.0)
-    jwt (1.5.6)
-    jwt_signed_request (2.4.0)
-      jwt (~> 1.5.0)
+    json (2.2.0)
+    jwt (2.1.0)
```

Comment:
### [jwt](http://github.com/jwt/ruby-jwt)

**Major** version upgrade :chart_with_upwards_trend::exclamation: 1.5.6 → 2.1.0

[_~~change-log~~, ~~source-code~~_]





<details>
<summary>Commits</summary>

A change of **151** commits. See the full changes on [the compare page](https://github.com/jwt/ruby-jwt/compare/v1.5.6...v2.1.0).

These are the first 10 commits:
- (2d2e924) [Bump version](https://github.com/jwt/ruby-jwt/commit/2d2e924c69f167e182628cbbc29ebf7a020c50a8)
- (83a8f43) [Fixed a typo in a spec name](https://github.com/jwt/ruby-jwt/commit/83a8f43d4cdb17b75f31247e70b29bc8fedb5e8a)
- (c234980) [Merge pull request #169 from Mingan/patch-1](https://github.com/jwt/ruby-jwt/commit/c23498058d801e444f228a64646c2fa21e05ad03)
- (c2304eb) [Pass payload to keyfinder.](https://github.com/jwt/ruby-jwt/commit/c2304eb5439fa7526213f07d5d4f6da4d8115ce5)
- (775c031) [Raise exception if keyfinder didn't return a key.](https://github.com/jwt/ruby-jwt/commit/775c031240f9fc4884b2ee83667496fcf4cfc9f9)
- (2f837b2) [Style changes from code review.](https://github.com/jwt/ruby-jwt/commit/2f837b259181fd0f14371e19a9d02f770b25ad07)
- (729f8db) [Merge pull request #172 from CodeMonkeySteve/master](https://github.com/jwt/ruby-jwt/commit/729f8db6d64940a3e92060034fe120adb472f298)
- (d83cca4) [Fix code style](https://github.com/jwt/ruby-jwt/commit/d83cca4ea14ae862391591167a21d99b7ed06739)
- (8e6c3f3) [Merge pull request #173 from excpt/master](https://github.com/jwt/ruby-jwt/commit/8e6c3f38b8ad4cd793420b4a5f729bc6ec96c6f5)
- (cc41d53) [Remove typ optional parameter](https://github.com/jwt/ruby-jwt/commit/cc41d53e00f8cbf015271b53f5ad761bd6ac2312)


</details>